Jenkinsfile

```diff
@@ -2,7 +2,7 @@
 /* groovylint-disable-next-line LineLength */
 /* groovylint-disable DuplicateMapLiteral, DuplicateNumberLiteral */
 /* groovylint-disable DuplicateStringLiteral, NestedBlockDepth, VariableName */
-/* Copyright 2019-2024 Intel Corporation
+/* Copyright 2019-2025 Intel Corporation
```

Comment:
Will need wait until #15678 or set manually
```suggestion
/* Copyright 2019-2024 Intel Corporation
/* Copyright 2025 Hewlett Packard Enterprise Development LP
```

Reply:
But I'm not touching this file